src/classes/Globals.js

```diff
@@ -50,14 +50,19 @@ class Globals {
 		this.initProps = {};
 
 		// domains
-		this.GHOSTERY_DOMAIN = this.DEBUG ? 'ghosterystage' : 'ghostery';
-		this.METRICS_SUB_DOMAIN = this.DEBUG ? 'staging-d' : 'd';
-		this.CMP_SUB_DOMAIN = this.DEBUG ? 'staging-cmp-cdn' : 'cmp-cdn';
-		this.CDN_SUB_DOMAIN = this.DEBUG ? 'staging-cdn' : 'cdn';
-		this.APPS_SUB_DOMAIN = this.DEBUG ? 'staging-apps' : 'apps';
-		this.GCACHE_SUB_DOMAIN = this.DEBUG ? 'staging-gcache' : 'gcache';
```

Comment:
@fcjr all these keep the `ghostery.com` domain even on staging, the stage vs prod difference is on the subdomain only.